pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -198,6 +199,10 @@ public class ModularLoadManagerImpl implements ModularLoadManager {
     private final Set<String> knownBrokers = new HashSet<>();
     private Map<String, String> bundleBrokerAffinityMap;
 
+    public static long nonActiveBundleDeleteThreshold = 15;
```

Comment:
why this threshold is needed ? sometimes a bundle is not loaded in this load report round. so a threshold is just to protect bundle not to be deleted for this case.

Reply:
Does that mean in the worst case, bundle deletion in the metadata store would take 15 mins in the worst case if the update cycle is every min?
Why have we picked 15?